coops-ui/src/components/Dashboard/DirectorListSm.vue

```diff
@@ -1,71 +1,114 @@
-      <v-avatar color="primary" size="25">
-        <span class="white--text small">{{ director.officer.firstName.substring(0,1)}}</span>
-      </v-avatar>
-      <div class="director-info">
+  <v-expansion-panels class="list list-container" :accordion=true :multiple=true>
+    <v-expansion-panel class="address-panel" v-for="director in directors" v-bind:key="director.id">
+      <v-expansion-panel-header>
+        <v-avatar color="primary" size="25">
+        <span class="">{{ director.officer.firstName.substring(0,1)}}</span>
+        </v-avatar>
         <div class="list-item__title">{{ director.officer.firstName }} {{ director.officer.lastName }}</div>
-        <div class="list-item__subtitle">
-          <ul class="address-details">
-            <li>{{ director.deliveryAddress.streetAddress }}</li>
-            <li>{{ director.deliveryAddress.addressCity }} {{ director.deliveryAddress.addressRegion }}
-              &nbsp;&nbsp;{{ director.postalCode}}</li>
-            <li>{{ director.deliveryAddress.addressCountry }}</li>
-            </ul>
-        </div>
-      </div>
-    </li>
-  </ul>
+      </v-expansion-panel-header>
+      <v-expansion-panel-content class="list-container-content">
+        <li class="list-item" >
+          <div class="director-info">
+            <div class="list-item_title">Delivery Address</div>
+            <div class="list-item__subtitle">
+              <ul class="address-details">
+                <li>{{ director.deliveryAddress.streetAddress }}</li>
+                <li>{{ director.deliveryAddress.addressCity }} {{ director.deliveryAddress.addressRegion }}
+                  {{ director.deliveryAddress.postalCode }}</li>
+                <li>{{ director.deliveryAddress.addressCountry }}</li>
+              </ul>
+            </div>
+          </div>
+        </li>
+        <li class="list-item" v-if="entityFilter(EntityTypes.BCorp)">
+          <div class="director-info">
+            <div class="list-item_title">Mailing Address</div>
+            <div class="list-item__subtitle">
+              <span v-if="isSameAddress(director.deliveryAddress, director.mailingAddress)">
+                Same as above
+              </span>
+              <ul v-else class="address-details" >
+                <li>{{ director.mailingAddress.streetAddress }}</li>
+                <li>{{ director.mailingAddress.addressCity }} {{ director.mailingAddress.addressRegion }}
+                  {{ director.mailingAddress.postalCode }}</li>
+                <li>{{ director.mailingAddress.addressCountry }}</li>
+              </ul>
+            </div>
+          </div>
+        </li>
+      </v-expansion-panel-content>
+    </v-expansion-panel>
+  </v-expansion-panels>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
+
+// Vue Libraries
+import { Component } from 'vue-property-decorator'
+import { mixins } from 'vue-class-component'
```

Comment:
Please use Mixins from vue-property-decorator instead. You can see an example of this in Directors.vue.

Reply:
Ok can do. What is the main reasoning?

Reply:
Consistency :)

I know that one package includes the other's functions, but another reviewer might question why it looks like we're using 2 different functions.

Reply:
Totally agree, makes perfect sense. Will update!